torch/_torch_docs.py

```diff
@@ -2915,14 +2919,16 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.isnan,
            r"""
+isnan(input) -> Tensor
+
 Returns a new tensor with boolean elements representing if each element is `NaN` or not.
 Complex values are considered `NaN` when either their real and/or imaginary part is NaN.
 
 Arguments:
-    input (Tensor): A tensor to check
+    {input}
```

Comment:
you are removing comments that input is a Tensor everywhere - is it universally assumed?

Reply:
You can see the autogenerated text and formatting for input styled like this in the doc for lgamma, for example: https://pytorch.org/docs/master/generated/torch.lgamma.html?highlight=lgamma#torch.lgamma. So yes, it is assumed.